ledger/src/blockstore_db.rs

```diff
@@ -314,7 +328,7 @@ impl Column for columns::TransactionStatus {
     type Index = (u64, Signature, Slot);
 
     fn key((index, signature, slot): (u64, Signature, Slot)) -> Vec<u8> {
-        let mut key = vec![0; 8 + 8 + 64];
+        let mut key = vec![0; 8 + 64 + 8];
```

Comment:
nit: a comment explaining these magic numbers would help current and future readers